MetaMask/metamask-extension
# chore: Adding installType to Sentry Tags for easy filtering

## **Description**
InstallType is a recently added flag to help quickly determine whether a Sentry issue is coming from a natural webstore install, or a developer environment. We want to be able to filter by this flag in the Sentry UI.

Added the tag, but also simplified some previous logic from when I added extensionId to make adding extra attributes less tedious in the future. We can also use this pattern for tags.


[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/28084?quickstart=1)

## **Related issues**

Fixes: #27667 

## **Manual testing steps**

1. Open App
2. Use developer options to trigger a sentry error
3. Go into Sentry UI and verify that installType is a tag in addition to being in the extra properties.

## **Screenshots/Recordings**
<img width="320" alt="Screenshot 2024-10-24 at 1 04 59 PM" src="https://github.com/user-attachments/assets/3ff828a5-c218-4222-9681-9001193380a0">


### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
